.release-it.json

```diff
@@ -0,0 +1,5 @@
+{
+  "hooks": {
+    "after:release": "ember deploy production"
```

Comment:
I am confused by [the docs](https://github.com/release-it/release-it#hooks). What is the difference between `afterRelease` and `after:release`?

Reply:
I honestly have no idea. Was just going to try this out, and if it didn't work we can tweak 😛